docs/UserGuide.md

```diff
@@ -157,8 +164,8 @@ Format: `add n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​ [w/WEDD
 </box>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/John Doe p/98765432 e/johnd@example.com a/XYZ Floral Services`
```

Comment:
Is this correct? Is XYZ Floral Services an address?

Reply:
or if we want to change the meaning of "address" to reflect the name of a company, maybe we should explicitly state it?